packages/store/addon/-private/system/snapshot-record-array.js

```diff
@@ -124,6 +124,14 @@ export default class SnapshotRecordArray {
   get type() {
     return this._type || (this._type = this._recordArray.get('type'));
   }
+  /**
+    The modelName of the underlying records for the snapshots in the array, as a DS.Model
+    @property type
+    @type {DS.Model}
+  */
+  get modelName() {
+    return this._recordArray.modelName;
```

Comment:
check whether `_type` here is the class or just an alias to modelName

Reply:
is class